src/Glpi/Form/Export/Serializer/FormSerializer.php

```diff
@@ -97,19 +99,64 @@ public function previewImport(
             $requirements = $form_spec->data_requirements;
             $mapper->mapExistingItemsForRequirements($requirements);
 
+            $form_id = $form_spec->id;
             $form_name = $form_spec->name;
+            if (in_array($form_id, $skipped_forms)) {
+                $results->addSkippedForm($form_id, $form_name);
+                continue;
+            }
+
             if ($mapper->validateRequirements($requirements)) {
-                $results->addValidForm($form_name);
+                $results->addValidForm($form_id, $form_name);
             } else {
-                $results->addInvalidForm($form_name);
+                $results->addInvalidForm($form_id, $form_name);
             }
         }
 
         return $results;
     }
 
+    public function resolveIssues(
```

Comment:
Can you rename this method ?
The current name seems to imply that it solve issues when it seems to be listing issues instead.